### DISCUSSION TITLE
My device (with 2FA for GitHub accunt) was stolen and don't have any recovery code.

### DISCUSSION THREAD

**Discussion by rifafauzi97** [community/community#68307] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Is there another way to login to my account with conditions like this and only relying on email, username and password? I already contacted GitHub Support, seems they don't have option for it. 

![Screenshot 2023-09-26 at 15 37 53](https://github.com/community/community/assets/115133791/d49bd113-92d7-4540-95d3-62585cbe9178)

Why? Why GitHub can't help us in this case? Why the last option is only to "Create new account" or "Unlink email and create new account using same email"?

I know I was careless because of not saving the recovery code, but Github should have another option to restore the account if 2FA (device) and any recovery code are lost. I am very disappointed because I can't recovery my account just only using username, email, and password

**Comment by rifafauzi97** [comment]
Hi @github please help me ^

**Comment by tuliogontijo** [comment]
I'm in the same situation! Lost my device and don't have my recovery codes. I tried to contact a human by chat, mail, phone, but found no way. I talk to a virtual assistant, but in the end it tells me to unlink my e-mail and fork the repositories I wanted. But I don't want it. There are lots of private repositories that I need and won't be able to fork.

Can anybody give me a solution

**Reply by GitlyHallows** [reply]
I have the same problem. I have unlinked the email address of my old account but I still need the old account back because of its private repositories.

**Comment by ghost** [comment]
Github is full of shit, take this as a learning opportunity and move to a different vendor like Gitlab.

**Comment by rifafauzi97** [comment, marked as answer]
I'm very lucky, when I installed the google authenticator again in my new device using same email as github account, the 2FA still there. So right now I can login again to my github account.
